elasticsearch-akka/src/main/scala/com/sumologic/elasticsearch/akkahelpers/ScanAndScrollSource.scala

```diff
@@ -49,7 +49,7 @@ class ScanAndScrollSource(index: Index, tpe: Type, query: QueryRoot, scrollSourc
   import akka.stream.actor.ActorPublisherMessage
 
   import ScanAndScrollSource._
-  import scala.concurrent.ExecutionContext.Implicits.global
+  implicit val ec = scrollSource.indexExecutionCtx
```

Comment:
not that it really matters to me, but shouldn't this be the search context?

Reply:
@rcoh Very interesting Q. At low level, it is performing a search. Though, scan&scroll is not intended to be used in the same thread pool as regular searches. It is a good point to raise though. It makes sense to use the ingest thread pool in our use case, but not in general case. Let me make it accept a parameter for scan and scroll. User can pass in which threadpool makes more sense in their use cases.

Reply:
Code changes for the above discussion here https://github.com/SumoLogic/elasticsearch-client/pull/136/commits/2350a42eb65ec7a134018ba1254b1f1b244f45ec